tbx/static_src/sass/config/_variables.scss

```diff
@@ -159,7 +220,7 @@ $tbxforms-secondary-text-colour: #505a5f; // Used for help text
 $tbxforms-border-colour-conditional: #b1b4b6;
 $tbxforms-input-border-colour: $tbxforms-text-colour();
 $tbxforms-hover-colour: #b1b4b6;
-$tbxforms-focus-colour: $focus-bg;
+$tbxforms-focus-colour: var(--color--focus-ring);
```

Comment:
I don't think we need any `$tbxforms` vars as it isn't in the build.

Reply:
Removed in https://github.com/torchbox/torchbox.com/pull/16/commits/ad5d02390efb8cbf63d63cd27f41b67c255c39fb